python/modules/SoftBDeepAK8SFProducer.py

```diff
@@ -64,6 +64,18 @@
     }
 }
 
+DeepTop_Fake_SF = {
+    "2016": {
+        (400, 9999) : (0.921, 0.066)
```

Comment:
I am a bit confused. I am expecting > 1 SF from the plot below. Even inclusive binning, the stat error bar indicating the 400-500 bin still have larger stat. 
![image](https://user-images.githubusercontent.com/1052165/75296267-4afcb900-57f2-11ea-955f-7c715a799979.png)

Reply:
The first and second bin have similar stats so the final single value is more or less its central average of the 2, why do you expect the average of these values in the plot to be more than 1?